### PR TITLE
PLANET-3712 add radio type

### PR DESCRIPTION
**Pull request by angtheod**
- Add radio field type.
- Auto-select field type for questions/optins as soon as they are added in the form.
- Retrieve for each locale the options (labels, values, selected) of the added question/optin.
- Show only the options of the selected locale.
- Fixed some bugs.
- Code cleanup.

### CODE REVIEW

File: includes/enform_post.twig [hunk 1]
```diff
@@ -111,6 +113,27 @@
 							</div>
 						</div>
 					{% endif %}
+				{% elseif 'radio' == field.input_type %}
```
Comment (kirdia) [inline includes/enform_post.twig]: @angtheod this clause is unreachable for questions or opt-ins

File: includes/selected_enform_fields.twig [hunk 2]
```diff
@@ -93,3 +94,29 @@
...
+					<option value="<%= label_option %>" <% print( label == label_option ?  'selected': '') %>><%= locale %></option>
+					<% }); %>
+				</select>
+			</div>
+			<br /><strong>{{ __( 'Question/Opt-in', 'planet4-engagingnetworks' ) }}</strong>
+			<br /><label class="question-label"></label><br /><br />
+			<% _.each( radio_options, function( radio_text, radio_value ) { %>
+			<input
+				id="en__field_supporter_{{ field.name }}"
+				type="radio"
```
Comment (kirdia) [inline includes/selected_enform_fields.twig]: should these display options for all locales?
Also radio type is not helpful here I think, values/options can't be unselected.

Comment (angtheod) [reply includes/selected_enform_fields.twig]: Nop, should display only for the selected locale. Fixing that in the following push.
I should disable the radio options anyway. Editor should not be able to check them.

File: includes/enform_post.twig (hunk 1 above)
Comment (angtheod) [reply includes/enform_post.twig]: Hmm are you sure? Why not?

**Comment by sagarsdeshmukh** [comment]
Hi @angtheod , could you please fix the conflicted files.

**Comment by angtheod** [comment]
@koyan @sagarsdeshmukh conflicts resolved

**Pull request merged**